### PR TITLE
Fixed translations for Serbian Latin.

### PR DESCRIPTION
**Pull request by stasavs**

### CODE REVIEW

File: src/sr_Latn/validation-inline.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
     'alpha_dash'           => 'Ovo polje može sadržati samo slova, brojeve i povlake.',
     'alpha_num'            => 'Ovo polje može sadržati samo slova i brojeve.',
     'array'                => 'Ovo polje mora sadržati nekih niz stavki.',
-    'attached'             => 'This field is already attached.',
+    'attached'             => 'Ovaj :attribute je već priložen.',
```
Comment (caouecs) [inline src/sr_Latn/validation-inline.php:12]: No :attribute here

**Review by caouecs — CHANGES_REQUESTED** [review]
Hello,
thank you, just a problem in validation-inline.php

**Comment by caouecs** [comment]
with the v9 of this package, the files are been moved

**Comment by andrey-helldar** [comment]
@stasavs, we have changed the structure of the project.

Files has been moved to a new location:

| old | new |
|:---|:---|
| `src/sr_Latn/auth.php` | `locales/sr_Latn/auth.php`|
| `src/sr_Latn/passwords.php` | `locales/sr_Latn/passwords.php`|
| `src/sr_Latn/validation-inline.php` | `locales/sr_Latn/validation-inline.php`|
| `src/sr_Latn/validation.php` | `locales/sr_Latn/validation.php`|

**Comment by andrey-helldar** [comment]
@stasavs, we hope you will come back to us 😊
The content of the files in our project remains the same, just the location has changed.

**Pull request closed without merging**